### PR TITLE
Allows partial regex for named parameters

### PR DESCRIPTION
**Pull request by gchauvet**
I must to re-create a merge request for #704 because my github repository was deleted, and some issues with my working copy.... all my apologies

### CODE REVIEW

File: lib/router.js [hunk 1]
```diff
@@ -248,10 +255,10 @@ Router.prototype.mount = function mount(options) {
     this.mounts[route.name] = route;
 
     this.emit('mount',
-        route.method,
-        route.path,
-        route.types,
-        route.versions);
+            route.method,
```
Comment (yunong) [inline lib/router.js]: @gchauvet Could you please revert these indentation changes in this file?

**Comment by yunong** [comment]
@gchauvet Thanks! Just one last thing, could you please revert the indentation changes in router.js?

File: lib/router.js (hunk 1 above)
Comment (gchauvet) [inline lib/router.js]: done ;)

**Comment by yunong** [comment]
Merged. Thanks so much for contributing.

**Pull request closed without merging**